client/components/site-preview-link/index.tsx

```diff
@@ -106,17 +107,24 @@ export default function SitePreviewLink( {
 	}
 	return (
 		<div>
+			<p>
+				{ translate(
+					"Sites in 'Coming Soon' mode are visible only to you and to the logged-in users you invite. Toggle 'Share site' to allow collaborators without an account to see your site. {{inlineSupportLink}}Learn more.{{/inlineSupportLink}}",
```

Comment:
Should `'`s be `"`s instead?

Reply:
Good point, yes, we should use double quotes.

Reply:
Updated with cb2165420297108adb1337107aad092f16230901